src/java/org/apache/cassandra/repair/SystemDistributedKeyspace.java

```diff
@@ -119,6 +123,15 @@ private SystemDistributedKeyspace()
                      + "status text,"
                      + "PRIMARY KEY ((keyspace_name, view_name), host_id))");
 
+    private static final TableMetadata BlacklistedPartitions =
```

Comment:
So in light of the discussions around how to do upgrades when there are different versions and different system_distributed tables, have we checked that this is gracefully handled if it is or isn't there (e.g. during a 4.0 -> 5.0 upgrade)?